src/Core/tests/DeviceTests.Shared/Stubs/ContextStub.cs

```diff
@@ -38,6 +38,12 @@ public object GetService(Type serviceType)
 
 			if (serviceType == typeof(NavigationRootManager))
 				return _windowManager ??= new NavigationRootManager(this);
+
+			if (serviceType == typeof(IFontManager))
```

Comment:
oh! do not need to do == against a the if you have the is operator
```suggestion
			if (serviceType is IFontManager)
```

Reply:
@jsuarezruiz don't want the suggestion?